modular-docs-manual/content/topics/module_modular-documentation-repositories.adoc

```diff
@@ -0,0 +1,6 @@
+[id="modular-documentation-repositories"]
+= Modular Documentation Repositories
+
+Due to many factors that determine documentation repository design for individual products, the modular documentation initiative cannot provide a strict documentation repository template for all documentation teams to follow. Each team faces different implementation challenges, especially when Red Hat enterprise product documentation is single-sourced with corresponding community project documentation.
```

Comment:
Keep it simple?
```suggestion
Due to many factors that determine repository design for individual products, these modular documentation guidelines do not provide a strict documentation repository template.
```

Reply:
Normally I'd agree, but I want to avoid being so concise and vague here. I want it crystal clear why we can't provide the strict template. Just saying "due to many factors" alone isn't sufficient, imo. So I'd prefer to keep the wording as is.

Reply:
I don't like "due to many factors", too, and wanted to cut even that...

My point is that it's a manual. Manuals should instruct while being as concise as possible -- not elaborate on reasons regarding the inclusion or exclusion of this or that. If you feel so strongly about it, I suggest we create an FAQ section at the end.

Reply:
Okay @rkratky , I give up :) For the sake of getting this through, I've taking your section here and below. Done.